client/components/Project/AddProjectComponent.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <button class="add-btn" @click="showModal">
+      + Add Project
+    </button>
+    <Modal
+      v-show="show"
+      :hideModal="hideModal"
+    >
+      <h2>Add Project</h2>
+      <form @submit.prevent="submit">
+        <div class="field">
+          <label for="project-name">
+            Name
+          </label>
+          <input
+            id="name"
+            name="name"
+            type="text"
+            placeholder="6.1040 Final Project"
```

Comment:
Is this a value that will always be the default value? Or can we make it a more generic "Project name" ?